### ISSUE TITLE
editor with 4 x map: waypoints do not correlate to map

### ISSUE THREAD

**Issue by Kuunlan01** [Stephan-S/FS19_AutoDrive#1171]
hey, 

I am trying to make this work on a 4x map called stone valley.
I converted the pda map file to a 204x2048png file but when I open the file in the editor the actual waypoints do not correlate with the map. I tried changing the width and height when converting but with little succes.
furterhermore: when I create waypoints somewhat near the map edges the waypoints do not show up in the editor.
see screenshot, where i drew the line is where there are still some wapoints but they are out of the viewing range.
Any suggestions? thanks in advance!

![image](https://user-images.githubusercontent.com/23531057/76700021-66513a80-66b3-11ea-9e47-bba073b94e62.png)

**Comment by raupach** [comment]
Did you try setting the zoom factor to 4x?

**Comment by Kuunlan01** [comment]
> Did you try setting the zoom factor to 4x?

thanks for he quick reply! I tried all zoomlevels :)

**Comment by raupach** [comment]
Can you provide the map(png-image) and your configuration so I can see what is happening there?

**Comment by Kuunlan01** [comment]
> Can you provide the map(png-image) and your configuration so I can see what is happening there?

I combined both in a zip file. i could upload the png file here

[config_and_png.zip](https://github.com/Stephan-S/FS19_AutoDrive/files/4334124/config_and_png.zip)

**Comment by Justice-Molton** [comment]
any news in this case? I've got the same problem

**Comment by raupach** [comment]
With the supplied data I can reproduce the problem. But what is going on with the data (config)? It seems to me that the coordinates in the config are a bit too big. So how exactly was this config created?

**Comment by Justice-Molton** [comment]
I would guess that it has to be with the fact, that it is a Config from a 4x map?

**Comment by Kuunlan01** [comment]
> With the supplied data I can reproduce the problem. But what is going on with the data (config)? It seems to me that the coordinates in the config are a bit too big. So how exactly was this config created?

Hey,
Thanks for looking into this.
I got the data from my savegame file.
I got this map from here: https://www.farming-simulator.com/mod.php?lang=en&country=us&mod_id=150974&title=fs2019
From what I read its a 4x but thinking about this issue I think it might be a custom size map?
Author of the map is called Lancyboi ( known FS contributor ), I can try to contact him?
Thanks again!,
